libs/ledger-live-common/src/families/cosmos/api/Cosmos.ts

```diff
@@ -312,4 +310,4 @@ export class CosmosAPI {
   };
 }
 
-export const defaultCosmosAPI = new CosmosAPI();
+export const defaultCosmosAPI = new CosmosAPI("cosmos");
```

Comment:
There shouldn't be a default API, I think we should remove this export.

Reply:
fixed